packaging/installer/kickstart.sh

```diff
@@ -1468,8 +1470,17 @@ try_package_install() {
   fi
 
   repoconfig_name="netdata-repo${release}"
-  repoconfig_file="${repoconfig_name}${pkg_vsep}${REPOCONFIG_VERSION}${pkg_suffix}.${pkg_type}"
-  repoconfig_url="${REPOCONFIG_URL_PREFIX}/${repo_prefix}/${repoconfig_file}/download.${pkg_type}"
+
+  case "${pkg_type}" in
+    deb)
+      repoconfig_file="${repoconfig_name}${pkg_vsep}${REPOCONFIG_DEB_VERSION}${pkg_suffix}.${pkg_type}"
+      repoconfig_url="${REPOCONFIG_DEB_URL_PREFIX}/${repo_prefix}/${repoconfig_file}/download.${pkg_type}"
+      ;;
+    rpm)
+      repoconfig_file="${repoconfig_name}${pkg_vsep}${REPOCONFIG_RPM_VERSION}${pkg_suffix}.${pkg_type}"
+      repoconfig_url="${REPOCONFIG_RPM_URL_PREFIX}/${repo_prefix}/${SYSARCH}/${repoconfig_file}"
+      ;;
+  esac
```

Comment:
I think we need to add `*` case + fatal (having anything except `deb` and `rpm` unexpected but possible due to a bug).

Reply:
From a completeness perspective I agree, but in practice this will only make the messaging in such a failure a bit friendlier to the user (we will still hit a reasonably ‘safe’ failure case very quickly without a default case).

Given the urgency of getting this merged, I would rather add this in a follow-up PR as opposed to requiring yet another round of testing.

Reply:
Mmm, (not handled) `*` case now:

 - `repoconfig_url` not set
 - `check_for_remote_file` fails
 - => _Could not install native binary packages, falling back to alternative installation method_

I am not sure that falling back to an alternative installation method is the desired result in this case.

Reply:
That is not technically possible with the current code (we always set `pkg_type` to either `deb` or `rpm`, but a typo or mistake in the future can be a problem).